lib/cloudinary.rb

```diff
@@ -140,18 +128,34 @@ def self.app_root
   end
 
   private
-  
+
+  def self.config_from_env
+    # Heroku support
+    if ENV["CLOUDINARY_CLOUD_NAME"]
+      config_keys = ENV.keys.select! { |key| key.start_with? "CLOUDINARY_" }
```

Comment:
I get the logic here, but I think it would be a bit safer to avoid passing arbitrary keys to the app. Makes room for configs that are invalid to get passed in without raising an error.

Reply:
I agree that it would be safer to whitelist only supported values, but that would require to have and maintain that list.
And it's not really different from what we have here:
https://github.com/cloudinary/cloudinary_gem/blob/aa9a05e4c47d6cba9ceead150cae1f10d0103f5c/lib/cloudinary.rb#L102

Reply:
Fair enough.